tests/python/mkl/test_mkldnn.py

```diff
@@ -121,47 +100,33 @@ def __getitem__(self, key):
 
     X = (32, 3, 32, 32)
     # trigger mkldnn execution thread
-    y = net(nd.array(np.ones(X))).asnumpy()
+    y = net(mx.nd.array(np.ones(X))).asnumpy()
 
     # Use Gluon dataloader to trigger different thread.
     # below line triggers different execution thread
     for _ in loader:
-        y = net(nd.array(np.ones(X))).asnumpy()
+        y = net(mx.nd.array(np.ones(X))).asnumpy()
         # output should have 0.3376348
-        assert_almost_equal(y[0, 0, 0, 0], 0.3376348)
+        assert_almost_equal(y[0, 0, 0, 0], 0.016711406)
```

Comment:
could you remind me what this is? why do you need to change it?
BTW, you need to update the comment as well.

Reply:
it just computes with fixed 1s as input, and this is the value i see as a result with GPU or cpu (non-mkldnn mode).